2-dockerize-wrapped-model/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.7-stretch AS builder
+
```

Comment:
Misschien een comment ter achtergrond van de twee images die worden aangesproken?
bijv. hier # copy app into regular image and install required libraries